streams/service/src/test/resources/custom-udf-microtest/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- Does not need a <parent> declaration -->
```

Comment:
Definitely yes.  If you give it a parent relationship, it tends to pull all the parent's dependencies into the jar (altho of course this can be worked around).  Also, as a test jar, it is intended to be as stand-alone as possible.

Finally, there's no benefit in declaring a parent, because it doesn't _need_ to inherit anything from streamline pom.  It's fine being just a sub-module for dependency-tree purposes.